client/src/containers/Explore.js

```diff
@@ -217,11 +239,73 @@ export const Explore = () => {
             .catch(error => console.log(error));    
     };
 
+    if (width < 840){ // mobile view
     return(
         <Wrapper>
         <Navbar />
         <main>
             <PageWrapper>
+                <AdBoxMobile>
+                    <adCarrier
+                    Height={50}
+                    Width={328}
+                    Path="cds_leaderboard"
+                    />
+                </AdBoxMobile>
```

Comment:
Here you can put the new AdBoxType component instead! And erase the duplicated code :)

```suggestion
                <AdBoxType>
                    <adCarrier
                    Height={50}
                    Width={328}
                    Path="cds_leaderboard"
                    />
                </AdBoxType>
```